code/modules/shuttle/computers/dropship_computer.dm

```diff
@@ -147,6 +147,11 @@
 		return
 
 	if(dropship_control_lost && skillcheck(user, SKILL_PILOT, SKILL_PILOT_EXPERT))
+		var/remaining_time = timeleft(door_control_cooldown) / 10
+		var/units = "seconds"
```

Comment:
redundant var just put it in the string

Reply:
i am not sure how i do that...
not even sure in what string you want me to put it....